src/assets/pages/blogposts/post-45.html

```diff
@@ -0,0 +1,33 @@
+<p><strong>. Universidade de Bras&iacute;lia (Unb) -</strong><span style="font-weight: 400;"> no &acirc;mbito da disciplina Evolu&ccedil;&atilde;o e Contribui&ccedil;&atilde;o em projetos de software, ministrada pela professora Carla Fonseca&nbsp;</span></p>
+<p><strong>. Centro Federal de Educa&ccedil;&atilde;o Tecnol&oacute;gica Celso Suckow da Fonseca (CEFET/RJ) -</strong><span style="font-weight: 400;"> no &acirc;mbito da disciplina Projeto e Constru&ccedil;&atilde;o de Sistemas, sob a orienta&ccedil;&atilde;o do professor Eduardo Bezerra</span></p>
+<p><strong>. Instituto Federal de Alagoas - </strong><span style="font-weight: 400;">dando continuidade ao </span><a href="https://exoonero.org/"><span style="font-weight: 400;">Projeto Exoonero</span></a><span style="font-weight: 400;">, orientado pelo professor Daniel Fireman</span></p>
+<p><strong>Desenvolvimento de artigo&nbsp;</strong></p>
+<p><span style="font-weight: 400;">Uma novidade do programa neste ciclo &eacute; o desenvolvimento de um artigo cient&iacute;fico orientado pela Cientista de Dados Lu&iacute;sa Coelho,&nbsp; pesquisadora bolsista pelo Programa Ci&ecirc;ncia da Dados Aplicadas &agrave; Sa&uacute;de (PCDaS/Fiocruz), parceira do programa Querido Di&aacute;rio nas Universidades.&nbsp;</span></p>
+<p><span style="font-weight: 400;">Juntamente com a estudante de Ci&ecirc;ncias Sociais na Universidade Federal de Pernambuco (UFPE), Nicolly Lira, Lu&iacute;sa vai desenvolver uma an&aacute;lise comparativa de t&eacute;cnicas de pr&eacute;-processamento de dados em textos de di&aacute;rios oficiais dos munic&iacute;pios inclu&iacute;dos na Rede Cegonha, programa governamental voltado para a sa&uacute;de de mulheres gr&aacute;vidas, entre os anos 2011 e 2021. Para esta pesquisa ser&atilde;o levantados dados utilizando o Querido Di&aacute;rio, ampliando o conhecimento desenvolvido em torno do projeto.&nbsp;</span></p>
+<p><strong>Querido Di&aacute;rio nas Universidades</strong></p>
+<p><span style="font-weight: 400;">Coordenado pelo Programa Ci&ecirc;ncia de Dados para Inova&ccedil;&atilde;o C&iacute;vica da Open Knowledge Brasil (OKBR) e apoiado pela PCDaS/Fiocruz, o programa est&aacute; no terceiro ciclo de atividades.&nbsp;</span></p>
+<p><span style="font-weight: 400;">Confira balan&ccedil;o dos ciclos anteriores:</span></p>
+<ul>
+<li><strong><a href="https://queridodiario.ok.org.br/blog/post/23"><strong>Querido Di&aacute;rio nas Universidades: iniciativa fecha semestre com resultados s&oacute;lidos</strong></a></strong></li>
+</ul>
+<ul>
+<li><a href="https://queridodiario.ok.org.br/blog/post/35"><strong>Querido Di&aacute;rio nas Universidades: confira os resultados do primeiro semestre de 2023</strong></a></li>
+</ul>
```

Comment:
Não precisam ser duas listas diferentes
```suggestion
<ul>
<li><strong><a href="https://queridodiario.ok.org.br/blog/post/23"><strong>Querido Di&aacute;rio nas Universidades: iniciativa fecha semestre com resultados s&oacute;lidos</strong></a></strong></li>
<li><a href="https://queridodiario.ok.org.br/blog/post/35"><strong>Querido Di&aacute;rio nas Universidades: confira os resultados do primeiro semestre de 2023</strong></a></li>
</ul>
```